e2e/ctl_v3_alarm_test.go

```diff
@@ -57,7 +57,8 @@ func alarmTest(cx ctlCtx) {
 	}
 
 	// make some space
-	if err := ctlV3Compact(cx, rev, true); err != nil {
+	// compact with the latest revision (store.currentRev.main)
+	if err := ctlV3Compact(cx, rev+2, true); err != nil {
```

Comment:
i do not understand this. we want to compact the keys put before line 47. they should occupy the majority of space. we need to figure out why compact them wont release space.